src/components-examples/material/input/input-harness/input-harness-example.html

```diff
@@ -0,0 +1,16 @@
+<mat-form-field>
+  <input matInput placeholder="Favorite food" value="Sushi" name="favorite-food">
+</mat-form-field>
+
+<mat-form-field>
+  <input matInput [type]="inputType"
+         [disabled]="disabled">
+</mat-form-field>
+
+<mat-form-field>
+  <textarea id="myTextarea" matInput placeholder="Leave a comment"></textarea>
+</mat-form-field>
+
+<mat-form-field>
+  <input matNativeControl placeholder="Native control" id="nativeControl">
```

Comment:
Some inputs are using `matInput` and others are going through `matNativeControl`. Maybe we should settle on one?